botfront/imports/ui/components/nlu/import-export/DataImport.jsx

```diff
@@ -32,12 +32,17 @@ export default class DataImport extends React.Component {
     }
 
     onDrop = (files) => {
+        const { model, instanceHost } = this.props;
         files.forEach((file) => {
             const reader = new FileReader();
-            reader.onload = () => this.setState({ values: reader.result });
+            reader.onload = () => {
+                Meteor.call('nlu.convertToJson', reader.result, model.language, 'json', instanceHost, wrapMeteorCallback((err, result) => {
+                    if (err) throw new Meteor.Error('Error in converting data', err);
```

Comment:
don't throw Meteor.error on front end and only update state if !err

Reply:
ok got it, will fix it